raycicmd/command_step.go

```diff
@@ -94,6 +94,9 @@ func (c *commandConverter) convert(step map[string]any) (
 	}
 
 	envMap := copyEnvMap(c.envMap)
+	if id != "" {
+		envMap["RAYCI_STEP_ID"] = id
```

Comment:
maybe `RAYCI_BUILDKITE_STEP_ID`, to avoid collision with some other logic happens to use the same name

Reply:
talked offline too. I don't want to introduce the buildkite concept here.

this is just informational for now. we can still change it (until it is being depended by something.. and we cannot..)